Changes in `osu! Philippines: CONQuest PH 2022` news post (#7704)
ppy/osu-wiki: news/2022-07-21-osu-philippines-conquest-ph-2022.md

Additionally, as a CONQuest PH 2022 exclusive, there will be a track from one of osu!'s featured artists, [LEAF XCEED Music Division](https://osu.ppy.sh/beatmaps/artists/88)! 

> For more information about the tournament proper, refer [here](/wiki/Tournaments/oPQS/1).

Reviewer: ```suggestion
For more information about the tournament, refer to [this wiki article](/wiki/Tournaments/oPQS/1).
```

## Prizes

> ### July 23, 2022 (Day 1)

Reviewer: can probably replace lines 25-39 with something like:

```md
*Note: The prizes are awarded for each day of the tournament.*

| Placing | Prizes |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, osu!merch, 14 months of osu!supporter |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | osu!merch, 4 months of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | osu!merch, 2 months of osu!supporter |
```

Author: Looks good tbh.
